src/static/js/manage_organization.js

```diff
@@ -1,3 +1,19 @@
+// Function to confirm deletion of an organization
+function confirmDelete() {
+    if (confirm("Are you sure you want to delete this organization?")) {
```

Comment:
I think this file has been updated. Can you please take the latest?

Reply:
weird, I didn't change these files, so I'll merge in from develop and just take develop branch as is @arsalananwar11